api/db/migrations/acquisition/001_create_acquisition_table.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS acquisition.acquisitions
+(
+  _id serial primary key,
+  application_id varchar NOT NULL,
```

Comment:
Pas sûr que l'on ai toujours une `application_id` pour les acquisitions qui sont, par exemple, importées via un fichier CSV dans l'interface du Registre.

Pourtant ça me semble bien d'être cohérents sur la logique d'importation et de considérer le module d'import CSV comme une application spécifique (auto-générée par opérateur s'il le faut).

On a un choix à faire :
1. module CSV = application, on garde le schema comme ça
2. action import CSV ≠ application, on enlève `NOT NULL` sur `application_id`

Reply:
Bon je vote pour 1 sans grande conviction et je ferme la conversation. Si jamais tu penses que 2 est mieux, dis le :)